ksqldb-tools/src/test/java/io/confluent/ksql/tools/migrations/MigrationsTest.java

```diff
@@ -266,6 +268,8 @@ private void shouldApplyMigrations() throws Exception {
         configFilePath,
         migrationsDir,
         "CREATE STREAM ${streamName} (A STRING) WITH (KAFKA_TOPIC='FOO', PARTITIONS=1, VALUE_FORMAT='JSON');\n" +
+            "ASSERT TOPIC FOO WITH (PARTITIONS=1) TIMEOUT 5 SECONDS;\n" +
+            "ASSERT NOT EXISTS SCHEMA ID 3 TIMEOUT 5 SECONDS;\n" +
```

Comment:
What would happen to a migration script if an assert failed?

Is that case worth testing?  (Admittedly, it may be a pain.)